tests/dags/test_only_empty_tasks.py

```diff
@@ -41,14 +41,14 @@ def __init__(self, body, *args, **kwargs):
 
 
 with dag:
-    task_a = DummyOperator(task_id="test_task_a")
+    task_a = EmptyOperator(task_id="test_task_a")
 
-    task_b = DummyOperator(task_id="test_task_b")
+    task_b = EmptyOperator(task_id="test_task_b")
 
     task_a >> task_b
 
-    task_c = MyDummyOperator(task_id="test_task_c", body={"hello": "world"})
+    task_c = MyEmptyOperator(task_id="test_task_c", body={"hello": "world"})
 
-    task_d = DummyOperator(task_id="test_task_on_execute", on_execute_callback=lambda *args, **kwargs: 1)
+    task_d = EmptyOperator(task_id="test_task_on_execute", on_execute_callback=lambda *args, **kwargs: 1)
 
-    task_e = DummyOperator(task_id="test_task_on_success", on_success_callback=lambda *args, **kwargs: 1)
+    task_e = EmptyOperator(task_id="test_task_on_success", on_success_callback=lambda *args, **kwargs: 1)
```

Comment:
This is going to fail on mypy test.
```
Run mypy.................................................................................Failed
- hook id: mypy
- exit code: 1

tests/dags/test_only_empty_tasks.py:54: error: Argument "on_execute_callback"
to "EmptyOperator" has incompatible type "Callable[[Context], int]"; expected
"Optional[Callable[[Context], None]]"  [arg-type]
    ...k_id="test_task_on_execute", on_execute_callback=lambda *args, **kwarg...
                                                        ^
tests/dags/test_only_empty_tasks.py:56: error: Argument "on_success_callback"
to "EmptyOperator" has incompatible type "Callable[[Context], int]"; expected
"Optional[Callable[[Context], None]]"  [arg-type]
    ...k_id="test_task_on_success", on_success_callback=lambda *args, **kwarg...
                                                        ^
Found 2 errors in 1 file (checked 1 source file)

ERROR: The previous step completed with error. Please take a look at output above 
```

In https://github.com/apache/airflow/pull/22832/commits/2a626c73270831ccc58d8e91f035868797dba697 I removed the init of EmptyOperator (see https://github.com/apache/airflow/pull/22832#discussion_r848155925 ). Setting it back resolves the mypy issue but I'm not sure if this is the way to go.

Reply:
Why do we have `**kwargs: 1` there ? I do not understand it - I believe it should be `**kwargs: None`  ?

Reply:
It's there since the test was added.
I'm not sure too but we both checked the PR 2 years ago :)
https://github.com/apache/airflow/pull/7880/files

Reply:
I changed it to `None` https://github.com/apache/airflow/pull/22974/commits/e1492dd6c399eef3ef0fc551d922030571edd7bb